src/main/kotlin/traffic_simulation/HelloWorld.kt

```diff
@@ -0,0 +1,6 @@
+fun main(args: Array<String>) {
+
+    println("Hello, World!")
+
```

Comment:
Eigentlich soll ein Test etwas sinnvolles testen. Im Rahmen dieses Issues sind Tests aber noch gar nicht notwendig, und selbst in #11 reicht es einen Test zu erstellen, der nichts aufruft außer `println`.